templates/ca.conf.j2

```diff
@@ -0,0 +1,20 @@
+
+[ ca ]
+default_ca = CA_default
+
+[ CA_default ]
+dir = {{ certificates_ca_key_dir }}
+certs = $dir/
+new_certs_dir = $dir/
+database = $dir/index.txt
+crlnumber = $dir/crl_number
+certificate = $dir/ca.crt
+private_key = $dir/ca-key.pem
+default_days = 3650
```

Comment:
It would be nice if I could configure the days

Reply:
fixed